redisdb/tests/test_default.py

```diff
@@ -71,6 +72,18 @@ def test_service_check(aggregator, redis_auth, redis_instance):
     assert sc.tags == ['foo:bar', 'redis_host:{}'.format(HOST), 'redis_port:6379', 'redis_role:master']
 
 
+@pytest.mark.integration
+def test_disabled_config_get(aggregator, redis_auth, redis_instance):
+    redis_check = Redis('redisdb', {}, {})
```

Comment:
Hmm. The Redis integration seems to be another example of a check enforcing the legacy A5 signature instead of passing `*args, **kwargs` through the base `AgentCheck`… (refs #5573)